includes/widgets/class-wc-widget-layered-nav.php

```diff
@@ -293,7 +293,7 @@ protected function layered_nav_dropdown( $terms, $taxonomy, $query_type ) {
 				if ( jQuery().selectWoo ) {
 					var wc_layered_nav_select = function() {
 						jQuery( '.dropdown_layered_nav_" . esc_js( $taxonomy_filter_name ) . "' ).selectWoo( {
-							placeholder: '" . wp_specialchars_decode( esc_js( $any_label ), ENT_QUOTES, get_bloginfo( 'charset' ) ) . "',
+							placeholder: '" . wp_specialchars_decode( esc_js( $any_label ), ENT_QUOTES ) . "',
```

Comment:
I recommend to change this to: `placeholder: " . wp_json_encode( (string)wp_specialchars_decode($any_label, ENT_QUOTES ) ) . ",`,  because example XSS for current code: `&amp;#x27+(alert(&amp;#x27Hello World!&amp;#x27),&amp;#x27&amp;#x27)+&amp;#x27`